boringtun/build.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/sh
 JNI_LIBS=../app/src/tun-blocka/jniLibs
 
-export PATH=$PATH:$NDK_STANDALONE/toolchains/llvm/prebuilt/linux-x86_64/bin
+# Change the next export as needed. It should point to your android ndk bundle
+export NDK=$HOME/Android/android-ndk-r21
```

Comment:
if you are fixing hardcoded paths maybe we should not assume that someone has $HOME/Android